app/src/main/java/uk/co/ribot/androidboilerplate/data/local/DatabaseHelper.java

```diff
@@ -102,8 +102,10 @@ private String createPlaceholders(int length) {
             throw new RuntimeException("No placeholders");
         } else {
             StringBuilder sb = new StringBuilder(length * 2 - 1);
-            sb.append("?");
-            for (int i = 1; i < length; i++) sb.append(",?");
+            sb.append('?');
+            for (int i = 1; i < length; i++) {
+                sb.append(",?");
```

Comment:
Yes, I don't think we allow single line for's. That's not on the guidelines. Plus PMD was saying is a bad thing to do.